server/src/main/java/io/crate/execution/engine/indexing/IndexWriterProjector.java

```diff
@@ -191,12 +191,14 @@ public String value() {
                     Maps.removeByPath(value, Arrays.asList(path));
                 }
             }
-            try (var stream = new BytesStreamOutput(lastSourceSize)) {
-                XContentBuilder xContentBuilder = new XContentBuilder(XContentType.JSON.xContent(), stream);
+            try (XContentBuilder xContentBuilder = new XContentBuilder(XContentType.JSON.xContent(), new BytesStreamOutput(lastSourceSize))) {
                 BytesReference bytes = BytesReference.bytes(xContentBuilder.map(value));
```

Comment:
closing content builder closes underlying stream as well because `Feature.AUTO_CLOSE_TARGET` is true by default.
But byte stream close in no-op anyway